src/app/profile/spaces/spaces.component.less

```diff
@@ -32,6 +31,12 @@
     overflow-y: scroll;
     width: 100%;
     margin-bottom: 0;
+
+    @media (min-width: @screen-lg-min) { height: 600px; }
```

Comment:
You can simplify this to be 
```
height: 600px;
@media (max-width: @screen-xs-min) { height: 500px; }
```
That way, it will always be 600px in height, until it hits the extra small resolution.

Reply:
Great, thanks Adam - I'll hop on these changes. Do the heights I put down make sense to you ? I test with the chrome inspector and everything looked pretty good to me, but I wanted to make sure with someone else.